esmf-samm-build-plugin/src/main/java/org/eclipse/esmf/samm/buildtime/diagram/DiagramRenderer.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (c) 2023 Robert Bosch Manufacturing Solutions GmbH
+ *
+ * See the AUTHORS file(s) distributed with this work for additional
+ * information regarding authorship.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+package org.eclipse.esmf.samm.buildtime.diagram;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import guru.nidi.graphviz.engine.Engine;
+import guru.nidi.graphviz.engine.Format;
+import guru.nidi.graphviz.engine.Graphviz;
+import guru.nidi.graphviz.model.MutableGraph;
+import guru.nidi.graphviz.parse.Parser;
+import net.sourceforge.plantuml.FileFormat;
+import net.sourceforge.plantuml.FileFormatOption;
+import net.sourceforge.plantuml.SourceStringReader;
+
+/**
+ * Mojo to render .dot (Graphviz) and .pu (PlantUML) diagrams to SVG. The created SVG will have the used font embedded.
+ */
+@Mojo( name = "renderDiagram", defaultPhase = LifecyclePhase.GENERATE_SOURCES )
+public class DiagramRenderer extends AbstractMojo {
+   private static final String FONT_FILE = "RobotoCondensed-Regular.ttf";
+   private static final String FONT_NAME = "Roboto Condensed";
+   private static final Pattern PRAGMA_LAYOUT_ENGINE = Pattern.compile( "^.*PRAGMA LAYOUT-ENGINE:\\s*(\\w+)$", Pattern.MULTILINE );
+   private final Logger LOG = LoggerFactory.getLogger( DiagramRenderer.class );
```

Comment:
I think, we can change the name LOG to log, because by naming convention variable names with Upper Case to be static.